### ISSUE TITLE
[FVT]:xcattest -c does not work for command list

### ISSUE THREAD

**Issue by tingtli** [xcat2/xcat-core#1223]
xCAT 2.12
xcattest does not work for command list.
```
c910f03c04k04:/ # xcattest -f linux.conf.template -c lsdef
xCAT automated test started at Fri May 27 04:01:15 2016
******************************
Reading Configure
******************************
OBJECT:c910f03c05k27,TYPE:node
    monserver = 10.3.4.4;
    vmnics = br10;
    currstate = install;
    serialport = 0;
    currchain = boot;
    profile = compute;
    vmcpus = 1;
    serialflow = hard;
    mgt = kvm;
    netboot = grub2;
    tftpserver = 10.3.4.4;
    statustime = 12-15-2015;
    serialspeed = 115200;
    primarynic = mac;
    vmnicnicmodel = virtio;
    mac = 42:dc:da:97:e5:ab;
    cons = kvm;
    groups = all;
    updatestatus = synced;
    vmstorage = dir:///var/lib/libvirt/images/;
    installnic = mac;
    updatestatustime = 08-01-2015;
    status = installing;
    ip = 10.3.5.27;
    nameservers = 10.3.4.4;
    conserver = 10.3.4.4;
    vmhost = c910f03c05;
    provmethod = sles12.1-ppc64le-install-compute;
    os = sles12.1;
    vmmemory = 4096;
    arch = ppc64le;
OBJECT:auto_test,TYPE:network
    mask = 10.0.0.0;
    gateway = 10.0.0.103;
    net = 10_0_0_0-255_0_0_0;
TABLE:passwd
    key = system
    username = root
    password = cluster


TABLE:site
    key = domain
    value = pok.stglabs.ibm.com


    key = forwarders
    value = 9.114.1.1


    key = nameservers
    value = 10.3.4.4


Varible:
    LATEST_VERSION = 2.12
    UBUNTU_MIGRATION2_CORE = https://xcat.org/files/xcat/xcat-core/2.11.x_Ubuntu/xcat-core/xcat-core-2.11.1-ubuntu.tar.bz2
    MaxLparID = 32
    MIGRATION2_DEP = http://xcat.org/files/xcat/xcat-dep/2.x_Linux/xcat-dep-2.11.1.tar.bz2
    MIGRATION1_DEP = http://xcat.org/files/xcat/xcat-dep/2.x_Linux/xcat-dep-2.10.tar.bz2
    CN = c910f03c05k27
    UBUNTU_MIGRATION1_CORE = https://xcat.org/files/xcat/xcat-core/2.10.x_Ubuntu/xcat-core/xcat-core-2.10.tar.bz2
    MN = c910f03c04k04
    UBUNTU_MIGRATION1_DEP = https://xcat.org/files/xcat/xcat-dep/2.x_Ubuntu/xcat-dep-ubuntu-20151012.tar.bz
    UBUNTU_MIGRATION2_DEP = https://xcat.org/files/xcat/xcat-dep/2.x_Ubuntu/xcat-dep-ubuntu-2.11.1.tar.bz2
    NETBOOTDIR = /opt/xcat/share/xcat/netboot/rh
    KITDATA = kitdata
    NETDRIVER = ibmveth
    MIGRATION1_VERSION = 2.10
    MIGRATION1_CORE = http://xcat.org/files/xcat/xcat-core/2.10.x_Linux/xcat-core/xcat-core-2.10.tar.bz2
    MIGRATION2_CORE = http://xcat.org/files/xcat/xcat-core/2.11.x_Linux/xcat-core/xcat-core-2.11.1-linux.tar.bz2
    MIGRATION2_VERSION = 2.11.1
    ISO = /iso/iso/suse/12/SLE-12-Server-DVD-ppc64le-GM-DVD1.iso
******************************
Initialize xCAT test evironment
******************************
chdef -t node -o c910f03c05k27 monserver=10.3.4.4 vmnics=br10 currstate=install serialport=0 currchain=boot profile=compute vmcpus=1 serialflow=hard mgt=kvm netboot=grub2 tftpserver=10.3.4.4 statustime=12-15-2015 serialspeed=115200 primarynic=mac vmnicnicmodel=virtio mac=42:dc:da:97:e5:ab cons=kvm groups=all updatestatus=synced vmstorage=dir:///var/lib/libvirt/images/ installnic=mac updatestatustime=08-01-2015 status=installing ip=10.3.5.27 nameservers=10.3.4.4 conserver=10.3.4.4 vmhost=c910f03c05 provmethod=sles12.1-ppc64le-install-compute os=sles12.1 vmmemory=4096 arch=ppc64le
chdef -t network -o auto_test mask=10.0.0.0 gateway=10.0.0.103 net=10_0_0_0-255_0_0_0
chtab key=system passwd.username=root passwd.password=cluster
chtab key=domain site.value=pok.stglabs.ibm.com
chtab key=forwarders site.value=9.114.1.1
chtab key=nameservers site.value=10.3.4.4
Detecting: OS = Linux
arch is ppc64le
Detecting: ARCH = ppc
mgt is kvm
Detecting: HCP = kvm
******************************
loading test cases
******************************
/opt/xcat/share/xcat/tools/autotest/testcase/addkit/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/addkitcomp/case0
/opt/xcat/share/xcat/tools/autotest/testcase/buildkit/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/chdef/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/chkkitcomp/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/chkosimage/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/chtab/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/chvm/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/confignics/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/copycds/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/dumpxCATdb/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/genimage/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/geninitrd/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/getmacs/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/gettab/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/imgexport/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/imgimport/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/installation/Diskless_installation_flat_p8_le
/opt/xcat/share/xcat/tools/autotest/testcase/installation/Full_installation_flat_p8_le
/opt/xcat/share/xcat/tools/autotest/testcase/installation/SN_setup_case
/opt/xcat/share/xcat/tools/autotest/testcase/installation/aix_diskless_installation
/opt/xcat/share/xcat/tools/autotest/testcase/installation/aix_full_installation
/opt/xcat/share/xcat/tools/autotest/testcase/installation/aix_statelite_installation
/opt/xcat/share/xcat/tools/autotest/testcase/installation/customize_sleep_for_sn
/opt/xcat/share/xcat/tools/autotest/testcase/installation/linux_cn_with_sn_diskless_installation_x86_64_redhat
/opt/xcat/share/xcat/tools/autotest/testcase/installation/linux_cn_with_sn_diskless_installation_x86_64_redhat_vm
/opt/xcat/share/xcat/tools/autotest/testcase/installation/linux_cn_with_sn_diskless_installation_x86_64_sles_vm
/opt/xcat/share/xcat/tools/autotest/testcase/installation/linux_cn_with_sn_full_installation_x86_64_redhat
/opt/xcat/share/xcat/tools/autotest/testcase/installation/linux_cn_with_sn_full_installation_x86_64_redhat_vm
/opt/xcat/share/xcat/tools/autotest/testcase/installation/linux_cn_with_sn_full_installation_x86_64_sles_vm
/opt/xcat/share/xcat/tools/autotest/testcase/installation/linux_cn_with_sn_statelite_installation_x86_64_redhat
/opt/xcat/share/xcat/tools/autotest/testcase/installation/linux_cn_with_sn_statelite_installation_x86_64_redhat_vm
/opt/xcat/share/xcat/tools/autotest/testcase/installation/linux_cn_with_sn_statelite_installation_x86_64_sles_vm
/opt/xcat/share/xcat/tools/autotest/testcase/installation/linux_diskless_installation
/opt/xcat/share/xcat/tools/autotest/testcase/installation/linux_diskless_installation_x86_64_redhat
/opt/xcat/share/xcat/tools/autotest/testcase/installation/linux_diskless_installation_x86_64_sles
/opt/xcat/share/xcat/tools/autotest/testcase/installation/linux_full_installation
/opt/xcat/share/xcat/tools/autotest/testcase/installation/linux_full_installation_x86_64_redhat
/opt/xcat/share/xcat/tools/autotest/testcase/installation/linux_full_installation_x86_64_sles
/opt/xcat/share/xcat/tools/autotest/testcase/installation/linux_sles_diskless_installation_ppc64
/opt/xcat/share/xcat/tools/autotest/testcase/installation/linux_sles_statelite_installation_ppc64
/opt/xcat/share/xcat/tools/autotest/testcase/installation/linux_sn_installation_ppc64_redhat
/opt/xcat/share/xcat/tools/autotest/testcase/installation/linux_sn_installation_x86_64
/opt/xcat/share/xcat/tools/autotest/testcase/installation/linux_sn_installation_x86_64_redhat
/opt/xcat/share/xcat/tools/autotest/testcase/installation/linux_sn_installation_x86_64_redhat_vm
/opt/xcat/share/xcat/tools/autotest/testcase/installation/linux_sn_installation_x86_64_sles
/opt/xcat/share/xcat/tools/autotest/testcase/installation/linux_sn_installation_x86_64_sles_vm
/opt/xcat/share/xcat/tools/autotest/testcase/installation/linux_statelite_installation
/opt/xcat/share/xcat/tools/autotest/testcase/installation/linux_statelite_installation_x86_64_redhat
/opt/xcat/share/xcat/tools/autotest/testcase/installation/linux_statelite_installation_x86_64_sles
/opt/xcat/share/xcat/tools/autotest/testcase/installation/litefile.csv
/opt/xcat/share/xcat/tools/autotest/testcase/installation/litefile_sles.csv
/opt/xcat/share/xcat/tools/autotest/testcase/installation/pre_deploy_sn
/opt/xcat/share/xcat/tools/autotest/testcase/installation/reg_linux_SN_installation_hierarchy
/opt/xcat/share/xcat/tools/autotest/testcase/installation/reg_linux_diskfull_installation_flat
/opt/xcat/share/xcat/tools/autotest/testcase/installation/reg_linux_diskfull_installation_flat_confignics_s
/opt/xcat/share/xcat/tools/autotest/testcase/installation/reg_linux_diskfull_installation_hierarchy
/opt/xcat/share/xcat/tools/autotest/testcase/installation/reg_linux_diskless_installation_flat
/opt/xcat/share/xcat/tools/autotest/testcase/installation/reg_linux_diskless_installation_hierarchy
/opt/xcat/share/xcat/tools/autotest/testcase/installation/reg_linux_statelite_installation_flat
/opt/xcat/share/xcat/tools/autotest/testcase/installation/reg_linux_statelite_installation_hierarchy
/opt/xcat/share/xcat/tools/autotest/testcase/installation/reg_linux_statelite_installation_hierarchy_by_nfs
/opt/xcat/share/xcat/tools/autotest/testcase/installation/reg_linux_statelite_installation_hierarchy_by_ramdisk
/opt/xcat/share/xcat/tools/autotest/testcase/installation/sles_full_installation_p8le
/opt/xcat/share/xcat/tools/autotest/testcase/installation/ubuntu_diskless_installation_cudafull
/opt/xcat/share/xcat/tools/autotest/testcase/installation/ubuntu_diskless_installation_cudaruntime
/opt/xcat/share/xcat/tools/autotest/testcase/installation/ubuntu_diskless_installation_vm
/opt/xcat/share/xcat/tools/autotest/testcase/installation/ubuntu_full_installation_cudafull
/opt/xcat/share/xcat/tools/autotest/testcase/installation/ubuntu_full_installation_cudaruntime
/opt/xcat/share/xcat/tools/autotest/testcase/installation/ubuntu_full_installation_vm
/opt/xcat/share/xcat/tools/autotest/testcase/installation/ubuntu_full_installation_vm_confignics_s
/opt/xcat/share/xcat/tools/autotest/testcase/installation/ubuntu_full_installation_vm_docker
/opt/xcat/share/xcat/tools/autotest/testcase/lsdef/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/lskit/case0
/opt/xcat/share/xcat/tools/autotest/testcase/lskitcomp/case0
/opt/xcat/share/xcat/tools/autotest/testcase/lskitdeployparam/case0
/opt/xcat/share/xcat/tools/autotest/testcase/lslite/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/lsvm/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/lsxcatd/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/makeconservercf/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/makedhcp/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/makedns/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/makehosts/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/makeknownhosts/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/makenetworks/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/makeroutes/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/mkdef/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/mkdsklsnode/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/mknimimage/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/mkvm/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/ngpfb/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/nodeadd/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/nodech/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/nodegrpch/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/nodels/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/noderange/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/noderm/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/nodeset/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/nodestat/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/packimg/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/pping/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/ppping/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/prsync/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/pscp/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/psh/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/regnotif/case0
/opt/xcat/share/xcat/tools/autotest/testcase/renergy/case_8233-E8B
/opt/xcat/share/xcat/tools/autotest/testcase/renergy/case_9125-F2C
/opt/xcat/share/xcat/tools/autotest/testcase/restapi/global/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/restapi/group/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/restapi/network/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/restapi/node/case0
/opt/xcat/share/xcat/tools/autotest/testcase/restapi/node/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/restapi/osimage/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/restapi/policy/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/restapi/service/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/restapi/table/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/restartxcatd/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/restorexCATdb/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/reventlog/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/rinv/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/rmdef/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/rmdsklsnode/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/rmkit/case0
/opt/xcat/share/xcat/tools/autotest/testcase/rmkitcomp/case0
/opt/xcat/share/xcat/tools/autotest/testcase/rmnimimage/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/rmvm/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/rpower/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/rscan/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/rspconfig/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/rvitals/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/sinv/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/sysclone/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/tabdump/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/tabgrep/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/tabprune/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/tabrestore/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/unregnotif/case0
/opt/xcat/share/xcat/tools/autotest/testcase/xcat2nim/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/xcatconfig/case0
/opt/xcat/share/xcat/tools/autotest/testcase/xcatsnap/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/xcatstanzafile/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/xdcp/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/xdsh/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/xdshbak/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/makentp/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/runcmdinstaller/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/switchdiscover/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/analyze_log/case0
/opt/xcat/share/xcat/tools/autotest/testcase/bmcdiscover/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/dockercommand/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/migration/redhat_migration
/opt/xcat/share/xcat/tools/autotest/testcase/migration/sles_migration
/opt/xcat/share/xcat/tools/autotest/testcase/migration/ubuntule_migration1_p8le
/opt/xcat/share/xcat/tools/autotest/testcase/migration/ubuntule_migration2_p8le
/opt/xcat/share/xcat/tools/autotest/testcase/migration/ubuntux_migration1_vm
/opt/xcat/share/xcat/tools/autotest/testcase/migration/ubuntux_migration2_vm
/opt/xcat/share/xcat/tools/autotest/testcase/pkg/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/updatenode/cases0
/opt/xcat/share/xcat/tools/autotest/testcase/xcatd/case0
To run:
lsdef_null
lsdef_a
lsdef_t_o_l
lsdef_t_o_l_z
lsdef_t
lsdef_t_i_o
lsdef_t_w
lsdef_s
lsdef_t_auditlog
lsdef_t_eventlog
lsdef_t_policy
lsdef_t_site
lsdef_t_err
lsdef_t_h_i
lsdef_nics
Not to run:
******************************
Start to run test cases
******************************



------Total: 0 , Failed: 0------

xCAT automated test finished at Fri May 27 04:01:19 2016
Please check results in the /opt/xcat/share/xcat/tools/autotest/result,
and see /opt/xcat/share/xcat/tools/autotest/result/failedcases.20160527040115
 file for failed cases.
see /opt/xcat/share/xcat/tools/autotest/result/performance.report.20160527040115
 file for time consumption
c910f03c04k04:/ # xcattest -h
Usage:xcattest - Run xcat test cases.
  xcattest [-?|-h]
  xcattest [-f configure file] [-b case bundle list] [--restore]
  xcattest [-f configure file] [-t case list] [--restore]
  xcattest [-f configure file] [-c cmd list] [-l] [--restore]
```

**Comment by hu-weihua** [comment]
@tingtli, I have fixed this issue by pull request #1237 . Could you please verify it? thanks

**Comment by tingtli** [comment]
I've verified with 6/27 build. xcattest -f linux.conf.template -c lsdef works fine.
